servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultSingleAddressHttpClientBuilder.java

```diff
@@ -737,11 +737,7 @@ protected void handleSubscribe(final Subscriber subscriber) {
 
         void nextError(final Throwable t) {
             seenError = true;
-            final CompletableSource.Processor oldProcessor = processor;
-            oldProcessor.onError(t);
-            final CompletableSource.Processor newProcessor = newCompletableProcessor();
-            newProcessor.onError(t);
-            processor = newProcessor;
```

Comment:
We discussed this in the past: https://github.com/apple/servicetalk/pull/1070#discussion_r434170797

If we do not replace the processor with the new one, we will hide newer exceptions because the original processor is already terminated. While you are here, please leave a comment to clarify that.

Reply:
ahh thx for reminding me. yes +1 to comment